wpilibc/src/main/native/include/Commands/CommandGroup.h

```diff
@@ -169,7 +169,7 @@ class CommandGroup : public Command {
   std::vector<CommandGroupEntry> m_commands;
 
   // The active children in this group (stored in entries)
-  std::list<CommandGroupEntry> m_children;
+  std::list<CommandGroupEntry*> m_children;
```

Comment:
Could this be done as a `std::vector` as well?  The only place where erase is used is in `CommandGroup::_Execute()`, and I think that could be rewritten to do a two-stage removal (e.g. loop over once like the current approach, but set removed elements to nullptr, then do a `std::remove_if` / `std::erase` pass to delete the nullptr elements as a second pass.  `std::list` is expensive compared to vector, particularly for small elements like pointers.

Reply:
Done.